cartridges/openshift-origin-cartridge-python/usr/versions/3.3-scl/lib/update-configuration

```diff
@@ -8,19 +8,19 @@ function update-configuration {
     # SCL installs of python 3.3
     sclpath=$(dirname $(scl enable python33 "which python"))
     env_dir="${OPENSHIFT_PYTHON_DIR}/env"
-    echo "$OPENSHIFT_PYTHON_DIR/virtenv/venv/bin:$OPENSHIFT_PYTHON_DIR/bin:${sclpath}" > $OPENSHIFT_PYTHON_DIR/env/OPENSHIFT_PYTHON_PATH_ELEMENT
+    echo "${OPENSHIFT_PYTHON_DIR}virtenv/bin:${OPENSHIFT_PYTHON_DIR}bin:${sclpath}" > ${env_dir}/OPENSHIFT_PYTHON_PATH_ELEMENT
```

Comment:
The 2.7 cartridges didn't have the extra venv folder in the dir. 

That would be a mistake on me.